### DISCUSSION TITLE
How to process a command

### DISCUSSION THREAD

**Discussion by dmtzs** [eternnoir/pyTelegramBotAPI#1532] [Q&A]
I want to create my own commands and send personalized messages and also execute other scripts but I dont understand how to do that, I need to host somewhere something or how it works?

**Comment by Badiboy** [comment]
https://pytba.readthedocs.io/en/latest/index.html

**Comment by dmtzs** [comment]
Thank you, also there's a section in which says how to add another user into a group channel?
